lms/djangoapps/grades/signals/handlers.py

```diff
@@ -133,6 +133,15 @@ def enqueue_subsection_update(sender, **kwargs):  # pylint: disable=unused-argum
             kwargs.get('points_possible'),
         )
     )
+    log.info(
+        u'Grades: Request async calculation of subsection grades. '
+        u'user_id:{}, course_key:{}, usage_key:{}, Task id: {}'.format(
+            kwargs['user_id'],
+            kwargs['course_id'],
+            kwargs['usage_id'],
```

Comment:
Because the purpose of the log is to identify that the task was created.  These are the fields required to identify which task it is.